packages/dotcom-server-handlebars/src/helpers/README.md

```diff
@@ -78,6 +78,16 @@ Example:
 
 [strictly equal]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness
 
+### ifEqualsSome
+
+Outputs the content if some of the parameters are [strictly equal].
```

Comment:
Could there be a note here saying that the first parameter will be used as a control and the remaining will be compared against it? Just to be a bit more explicit.

Reply:
Added this for ifEquals as well as ifEqualsSome, just to make it crystal clear on the functionality of both!